src/client/app/components/ExportComponent.tsx

```diff
@@ -50,6 +53,10 @@ export default function ExportComponent(props: ExportProps) {
 		const name = `oedExport_${chartName}_${startTimeString}_to_${endTimeString}.csv`;
 		graphExport(compressedData,	name);
 	};
+	/*
+	 * TODO: I am noticing that some files have specific "render" methods for the JSX code, 
+	 * while others just return it. Is there a reason for this inconsistency I am missing?
+	 */
```

Comment:
The difference is between functional and class-based components. https://reactjs.org/docs/components-and-props.html